examples/drain3.ini

```diff
@@ -16,8 +16,9 @@ mask_prefix = <:
 mask_suffix = :>
 
 [DRAIN]
+matched_pattern = JaccardDrain
 sim_th = 0.4
-depth = 4
+depth = 3
```

Comment:
Again, we should not change the default behavior unless there is a good reason.

Reply:
ok, default behavior same as before is better.

Reply:
ok, default behavior same as before is better.